paddle/fluid/memory/allocation/mmap_allocator.cc

```diff
@@ -30,6 +30,9 @@ namespace paddle {
 namespace memory {
 namespace allocation {
 
+thread_local std::tuple<size_t, std::map<void *, size_t>> allocated_memory(
```

Comment:
Better rename to mmaped_memory and move it into RefcountedMemoryMapAllocation as static member. And, as member, I think tuple is not needed.